backend/manager/modules/bll/src/main/java/org/ovirt/engine/core/bll/storage/connection/ISCSIStorageHelper.java

```diff
@@ -207,6 +207,14 @@ public StorageServerConnections findConnectionWithSameDetails(StorageServerConne
         return null;
     }
 
+    public List<StorageServerConnections> findConnectionByAddressPortAndIqn(StorageServerConnections connection) {
```

Comment:
Maybe rename to `findConnectionsByAddressPortAndIqn`?
Add "s" for plural...

Reply:
Ack